debian/changelog.in

```diff
@@ -0,0 +1,5 @@
+nix (@PACKAGE_VERSION@-1) unstable; urgency=low
+
+  * Initial release.
+
+ -- Eelco Dolstra <eelco.dolstra@logicblox.com>  Sun, 13 Dec 2009 00:11:46 +0300
```

Comment:
Should this date be the date of the initial release or the data of PACKAGE_VERSION release?